cli/commands/genesis/payload.go

```diff
@@ -97,7 +97,7 @@ func AddExecutionPayloadCmd(chainSpec chain.Spec) *cobra.Command {
 				executableDataToExecutionPayloadHeader(
 					genesisInfo.ForkVersion.ToUint32(),
 					payload,
-					chainSpec.MaxWithdrawalsPerPayload(),
+					chainSpec.MaxWithdrawalsPerPayload(0),
```

Comment:
```suggestion
					chainSpec.MaxWithdrawalsPerPayload(constants.GenesisSlot),
```
(along with the import for the file)